importer/Import/Repodata.hs

```diff
@@ -94,7 +94,11 @@ loadRepoFromURI uri = do
 
  where
     appendOrThrow :: T.Text -> URI
-    appendOrThrow path = appendURI uri (T.unpack path) `throwIfNothing` RepoException
+    appendOrThrow path = let
+          -- If the URI path already ends in a slash, don't add an extra one
+          new_path = if last (uriPath uri) == '/' then T.unpack path else "/" ++ T.unpack path
```

Comment:
Steps are in the first comment, just try to import from a repository URL which doesn't end in a trailing slash. The importer is supposed to append the repodata/repomd.xml to that URL and figure out where primary.xml.gz is. In the above example if you add a trailing slash to the URL you can see packages being imported.